.expeditor/config.yml

```diff
@@ -143,3 +143,6 @@ subscriptions:
   - workload: ruby_gem_published:cheffish-*
     actions:
       - bash:.expeditor/update_dep.sh
+  - workload: ruby_gem_published:license-acceptance-*
```

Comment:
Can you comment these subscriptions all out for now actually. They're not working and just create a ton of noise in the project.